homeassistant/components/somfy_mylink/__init__.py

```diff
@@ -92,19 +99,20 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
             "Unable to connect to the Somfy MyLink device, please check your settings"
         ) from ex
 
-    if "error" in mylink_status:
+    if not mylink_status or "error" in mylink_status:
```

Comment:
What is the scenario for this being false? And how will the `["error"]` access below work in such a case?

Reply:
I've received an empty response from the device every so often.  I think its a bug in the device itself.

Reply:
I reverted the `.get(` change

Reply:
I added text for when we get an empty response seen on firmware 5.23